bubblesub/ui/statusbar.py

```diff
@@ -53,6 +54,20 @@ def __init__(
             self._on_audio_selection_change
         )
 
+    def set_visibility(self, current_view_layout: ViewLayout) -> None:
```

Comment:
Let's create a signal in the view manager that the layout has changed and listen to it here